aea/crypto/wallet.py

```diff
@@ -70,8 +70,18 @@ def addresses(self) -> Dict[str, str]:
 
 
 class Wallet:
-    """Store all the cryptos we initialise."""
+    """
+    Container for crypto objects.
 
+    The cryptos are separated into two categories:
+
+    - main cryptos: used by the AEA for the economic side (i.e. signing transaction)
+    - connection cryptos: exposed to the connection objects for encrypted communication.
+
+    """
+
+    # TODO do some check after loading the keys
+    #   to see whether we have duplicate cryptos?
```

Comment:
Yes; though currently only duplication across crypto stores is possible not within due to unique keys in private key paths dict. We should at most provide a warning if there is duplication across cryptos (as it might be intended)

Reply:
ok